core/domain/question_services.py

```diff
@@ -309,7 +393,9 @@ def update_skill_ids_of_questions(
     old_question_skill_links = get_question_skill_links_of_skill(
```

Comment:
Just to check, does old_question_skill_link_models return a complete list, or just the first N questions (if there are a lot of questions tied to this skill)? I think we normally have built-in limits so you might need to think about ensuring that this function actually retrieves _all_ the models that need changing. Might be worth writing a test that tries to do this with 1000 models.

Reply:
I haven't put any limits but is there a way to retrieve all models? (Like is there a flag?)

Reply:
I couldn't find anything about it but I'll investigate further